WordPress/WordPressTest/NSManagedObject+Fixture.swift

```diff
@@ -0,0 +1,24 @@
+import CoreData
+
+extension NSManagedObject {
+
+    /// Loads the contents of any given JSON file into a new NSManagedObject instance.
+    ///
+    /// This helper method is useful for Unit Testing scenarios.
+    ///
+    /// - Parameters:
+    ///   - filename: The name of the JSON file to be loaded
+    ///   - context: The managed object context to use
+    /// - Returns: A new instance with property values of the given JSON file.
+    static func fixture(fromFile filename: String, context: NSManagedObjectContext) -> Self {
+        guard let jsonObject = JSONLoader().loadFile(named: filename) else {
+            fatalError("Mockup data could not be parsed, the filename is \(filename)")
```

Comment:
Have you considered the alternative of returning `nil` from here?

Reply:
I kept the old behaviour which was crashing via `NSAssert`, but as mentioned in [this comment](https://github.com/wordpress-mobile/WordPress-iOS/pull/18478#discussion_r862666547), I'll do it properly this time to make this function throw.